openwisp_network_topology/settings.py

```diff
@@ -2,7 +2,10 @@
 
 from django.conf import settings
 
-logger = logging.getLogger(__name__)
+# The logger 'pre_django_setup' is suppose to be
+# a logger that can work before registered
+# Apps are ready in django.setup() process.
+logger = logging.getLogger('pre_django_setup')
```

Comment:
The root logger may contain handler `mail_admin`, which has class `'class': 'django.utils.log.AdminEmailHandler'` which needs apps to be registered before usage.

However, this can be called before Apps are registered, which can cause `django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.`  in order to avoid this, I pull a different logger name, which the user can define in such a way that App registration is not required. 

P.S: If `pre_django_setup` is not defined, it fallls back to root logger, hence this is backward compatible.